library/src/main/kotlin/app/futured/donut/DonutProgressView.kt

```diff
@@ -14,6 +21,7 @@ import androidx.annotation.ColorInt
 import androidx.core.animation.doOnEnd
 import androidx.core.content.ContextCompat
 import androidx.core.content.res.use
+import androidx.core.graphics.flatten
 import app.futured.donut.extensions.sumByFloat
 
 /*
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/thefuntasty/donut/blob/961d58caf19d5189c8ff62ff1a0576433dabdd98/library/src/main/kotlin/app/futured/donut/DonutProgressView.kt#L27"></span>Class DonutProgressView is too large. Consider splitting it into smaller pieces.</td>
    </tr>
  </tbody>
</table>